packages/graphql/tests/integration/aggregations/where/edge/bigint.int.test.ts

```diff
@@ -0,0 +1,378 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { generate } from "randomstring";
+import neo4j from "../../../neo4j";
+import { Neo4jGraphQL } from "../../../../../src/classes";
+
+describe("aggregations-where-edge-bigint", () => {
+    let driver: Driver;
+
+    const bigInt = "2147483647";
+
+    beforeAll(async () => {
+        driver = await neo4j();
+    });
+
+    afterAll(async () => {
+        await driver.close();
+    });
+
+    test("should return posts where a edge like BigInt is EQUAL to", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+            type User {
+                testString: String!
+            }
+          
+            type Post {
+              testString: String!
+              likes: [User] @relationship(type: "LIKES", direction: IN, properties: "Likes")
+            }
+
+            interface Likes {
+                someBigInt: BigInt
+            }
+        `;
+
+        const testString = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        try {
+            await session.run(
+                `
+                    CREATE (:Post {testString: "${testString}"})<-[:LIKES {someBigInt: ${bigInt}}]-(:User {testString: "${testString}"})
```

Comment:
`someBigInt` won't be an integer in the db here I believe.
You should have it as a cypher parameter and wrap it in `neo4j.int(bigInt)`.

Reply:
Actually, you could probably use the Cypher function `toInteger()` on a number to convert it without using a param.

Reply:
Good observation